vignettes/anatomy.Rmd

```diff
@@ -89,17 +82,13 @@ Putting all these together (by explicitly setting the default config in the `glo
 log_threshold(INFO)
 log_formatter(formatter_glue)
 log_layout(layout_simple)
-log_appender(appender_console)
+log_appender(appender_stdout)
```

Comment:
This was causing log output to be lost since knitr doesn't (and IMO shouldn't) capture stderr.